en-US/Players_Guide/03_Creating_a_Character/Languages.md

```diff
@@ -1,11 +1,11 @@
 ### Languages
 
-Your culture determines your default languages that you can read, speak, write, and sign, provided there is no disability or condition that prevents you from doing so.
+Your culture determines your default languages that you can speak, read, and write, provided there is no disability or condition that prevents you from doing so. Some characters may also know one or more signed languages. All Standard Languages have a separate signed analog.
 Your background may grant access to additional languages of your choice.
 Note these languages on your character sheet.
 
 Choose from the Standard Languages table, or a common language in your campaign.
-With your Conductor’s permission, you can choose an exotic language if it fits your background.
+With your Conductor’s permission, you can choose an exotic language if it fits your background. Your Conductor can decide which Exotic Languages also have signed analogs.
```

Comment:
Please add the new sentence in a new line. That's how it's done everywhere else in this repository.